lib/cli.js

```diff
@@ -38,6 +38,7 @@ exports.run = () => {
 		  --verbose, -v           Enable verbose output
 		  --no-cache              Disable the transpiler cache
 		  --no-power-assert       Disable Power Assert
+		  --no-color              Disable color output
```

Comment:
This flag should be added to readme.md (like -no-power-assert for example) in [CLI](https://github.com/avajs/ava/blob/master/readme.md#cli) paragraph